EventFiltering/PWGHF/HFFilter.cxx

```diff
@@ -179,7 +179,10 @@ struct HfFilter { // Main struct for HF triggers
 
     hProcessedEvents = registry.add<TH1>("fProcessedEvents", "HF - event filtered;;counts", HistType::kTH1F, {{kNtriggersHF + 2, -0.5, +kNtriggersHF + 1.5}});
     for (auto iBin = 0; iBin < kNtriggersHF + 2; ++iBin) {
-      hProcessedEvents->GetXaxis()->SetBinLabel(iBin + 1, eventTitles[iBin].data());
+      if (iBin < 2)
+        hProcessedEvents->GetXaxis()->SetBinLabel(iBin + 1, eventTitles[iBin].data());
```

Comment:
maybe we can remove the entries in `eventTitles` beyond the second one, just to avoid confusion?